application/controllers/admin/Translate.php

```diff
@@ -215,6 +215,9 @@ private function displayUntranslatedFields($iSurveyID, $tolang, $baselang, $tab_
                     $aResultTo2 = !empty($type2) ? $oResultTo2->questiongroupl10ns[$tolang]->getAttributes() : $aResultTo;
                 } elseif ($class == 'Question' || $class == 'Subquestion') {
                     $aRowfrom = $oRowfrom->questionl10ns[$baselang]->getAttributes();
+                    if (!empty($oRowfrom['parent_qid'])) {
+                        $aRowfrom['parent'] = $oRowfrom->parent->getAttributes();
```

Comment:
Maybe (for broken DB) check if parent exist ? 

`if (!empty($oRowfrom['parent_qid']) and !empty($oRowfrom->parent)) {`

I know it must be fixed by Check_data_integrity, then maybe leave it ?